openhands/server/routes/git.py

```diff
@@ -93,9 +96,14 @@ async def get_github_installation_ids(
 ):
     if provider_tokens and ProviderType.GITHUB in provider_tokens:
         token = provider_tokens[ProviderType.GITHUB]
+        # Use the host_url from the token if available, otherwise use the default API URL
+        github_api_url = token.host_url or get_github_api_url()
```

Comment:
We should have host_url information available in the token, so `get_github_api_url` should be unnecessary

We strictly want to store such information on a per user basis